packages/cyberstorm-styles/src/styles/theme.css

```diff
@@ -31,4 +31,6 @@
   --color-text--default: var(--color-lilac--lightest);
   --color-text--secondary: var(--color-lilac--lighter);
   --color-text--tertiary: var(--color-lilac--light);
+  --animation-duration-1: 400ms;
```

Comment:
--animation-duration-default or --animation-duration-medium would be better imo (then in the future we could add slower and faster verisions)

Reply:
Valid point. Fixed it to be *-default 👌